features/src/main/feature/camel-features.xml

```diff
@@ -1154,66 +1154,52 @@
-<!--        <bundle>mvn:io.fabric8/openshift-model/${kubernetes-model-version}</bundle>-->
-<!--        <bundle>mvn:io.fabric8/openshift-model-console/${kubernetes-model-version}</bundle>-->
-<!--        <bundle>mvn:io.fabric8/openshift-model-clusterautoscaling/${kubernetes-model-version}</bundle>-->
-<!--        <bundle>mvn:io.fabric8/openshift-model-hive/${kubernetes-model-version}</bundle>-->
-<!--        <bundle>mvn:io.fabric8/openshift-model-machine/${kubernetes-model-version}</bundle>-->
-<!--        <bundle>mvn:io.fabric8/openshift-model-machineconfig/${kubernetes-model-version}</bundle>-->
-<!--        <bundle>mvn:io.fabric8/openshift-model-monitoring/${kubernetes-model-version}</bundle>-->
-<!--        <bundle>mvn:io.fabric8/openshift-model-miscellaneous/${kubernetes-model-version}</bundle>-->
-<!--        <bundle>mvn:io.fabric8/openshift-model-operator/${kubernetes-model-version}</bundle>-->
-<!--        <bundle>mvn:io.fabric8/openshift-model-operatorhub/${kubernetes-model-version}</bundle>-->
-<!--        <bundle>mvn:io.fabric8/openshift-model-storageversionmigrator/${kubernetes-model-version}</bundle>-->
-<!--        <bundle>mvn:io.fabric8/openshift-model-tuned/${kubernetes-model-version}</bundle>-->
-<!--        <bundle>mvn:io.fabric8/openshift-model-whereabouts/${kubernetes-model-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:io.fabric8/kubernetes-client/${kubernetes-client-version}/jar/bundle</bundle>-->
-<!--        <bundle dependency='true'>mvn:io.fabric8/openshift-client/${kubernetes-client-version}/jar/bundle</bundle>-->
-<!--        <bundle>mvn:org.apache.camel/camel-kubernetes/${project.version}</bundle>-->
-<!--    </feature>-->
+    <feature name='camel-kubernetes' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-core</feature>
+        <feature version='${jackson2-version}'>jackson</feature>
+        <bundle dependency='true'>mvn:io.fabric8/kubernetes-model-apiextensions/${kubernetes-model-version}</bundle>
+        <bundle dependency='true'>mvn:io.fabric8/kubernetes-model-apps/${kubernetes-model-version}</bundle>
+        <bundle dependency='true'>mvn:io.fabric8/kubernetes-model-autoscaling/${kubernetes-model-version}</bundle>
+        <bundle dependency='true'>mvn:io.fabric8/kubernetes-model-batch/${kubernetes-model-version}</bundle>
+        <bundle dependency='true'>mvn:io.fabric8/kubernetes-model-certificates/${kubernetes-model-version}</bundle>
+        <bundle dependency='true'>mvn:io.fabric8/kubernetes-model-coordination/${kubernetes-model-version}</bundle>
+        <bundle dependency='true'>mvn:io.fabric8/kubernetes-model-core/${kubernetes-model-version}</bundle>
+        <bundle dependency='true'>mvn:io.fabric8/kubernetes-model-common/${kubernetes-model-version}</bundle>
+        <bundle dependency='true'>mvn:org.apache.aries.spifly/org.apache.aries.spifly.dynamic.bundle/1.3.7</bundle>
```

Comment:
`org.apache.aries.spifly.dynamic.bundle` is required because inside `kubernetes-model-common` bundle, we have this declaration:
```
 Require-Capability: osgi.extender;filter:="(osgi.extender=osgi.service
 loader.registrar)"
```

Reply:
To fix this kind of issue, you need to depend on spifly like I did here https://github.com/jbonofre/camel-karaf/commit/8ad9a42ec0a6bdf8a384c0387c46db4b5a6162de#diff-8cca327d8104823fc5bc3583bb826255c61052b3b902b59d0b1ba5bc0fe9342eR1700

Reply:
I applied the suggestion in my last commit 👍